pkg/constants/constants.go

```diff
@@ -51,8 +51,8 @@ const (
 	// cluster are in maintenance mode.
 	MaintenanceAnnotationLabel = "capv." + v1alpha3.GroupName + "/maintenance"
 
-	// DefaultEnableKeepAlive is false by default.
-	DefaultEnableKeepAlive = false
+	// DefaultEnableKeepAlive is true by default.
+	DefaultEnableKeepAlive = true
```

Comment:
Just for other reviewers. Please note this is not a change in behavior as the default value  / flag was effectively not used anymore